src/components/header/_macro.njk

```diff
@@ -102,9 +102,9 @@
                                                         href="{{ item.url }}"
                                                         class="ons-header-service-nav__link"
                                                         {% if item.id %} id="{{ item.id }}"{% endif %}
-                                                    >{{ item.title }}</a>
+                                                    >{% if item.title %}{{ item.title }}{% elif item.text %}{{ item.text }}{% endif %}</a>
```

Comment:
Is this to make the changes not breaking?

Reply:
Correct :)

Reply:
Is this a good idea? Leaving legacy code in? Personally I feel like it would be better to just make the breaking change. Your call though.

Reply:
That was my initial thought yeah, however since it's in the header, I think it would affect a lot of users. Had a chat with Rich about it who raised the ticket initially and he thinks it's better to avoid a BC if we can for a time. Raised another ticket as a reminder to take it out later too.